### ISSUE TITLE
Alternatives to "×××" under Filtering > Block

### ISSUE THREAD

**Issue by bearsomehand** [FrankerFaceZ/FrankerFaceZ#1444]
**Is your feature request related to a problem? Please describe.**
After blocking more than 1 emote/term, it becomes impossible to distinguish between blocked emotes/terms as they are _>all<_ displayed as "**_×××_**" in the chat.

**Describe the solution you'd like**
An option to individually choose what is displayed instead of the blocked emote/term. I.E. a different one for each individual emote/term, I would prefer any text including the alphabet, <, [, (, /, dots and other lines, spaces etc. (and even other emotes if possible).  "**_×××_**" could still be the default(1). _Edit: I'd also keep it in italic(and bold I presume?) and/or have the option to use italic/bold/strike through line, a line under the text, etc._

**Describe alternatives you've considered**
Could be as simple as the ability to chose different numbers of "**_×_**". _Example: "**_×_**" for one emote/term, "**_××_**" for another etc._

**Additional context**
I have _-insert diagnosis-_ and strongly dislike >physically seeing< certain emotes, blocking several emotes/terms means that I am unable to discern what is being communicated in chat...which IMO defeats the purpose of reading the chat.

In other words: this feature would greatly improve my chat experience as I like to know what other chatters are communicating and/or what their reaction to something is while avoiding physically seeing certain emotes.

It would also give everyone the option to distinguish between umbrella concepts for their blocks, such as profanity vs adult themed or text vs emotes etc. _(without having to see the specific blocked emote/term)_ by chosing 1 representation for all blocked emotes/terms they personally think fit a concept~

_(1)Keeping "**×××**" as the default means that people who >don't< want to know which blocked emote/term is being used won't be affected by this feature and it doesn't make it any more confusing/complicated for them._

**Comment by Lordmau5** [comment]
There's now a new add-on called "No Bad Emotes" which implements a functionality to prevent words to turn into emotes

I've posted some screenshots here #1321 

I do assume that is taking care of this specific feature request? In which case I'll go ahead and close the issue :)
